### DISCUSSION TITLE
Dex follower only, sensor always shows started

### DISCUSSION THREAD

**Discussion by mkitchingh** [NightscoutFoundation/xDrip#2001] [Q&A] [answered]
I use xDrip+ to follow my daughter only. The data source is Dexcom Share. It used to be extremely sluggish and would frequently show messages saying it was not responding. I saw a post in here (FB group) once about something similar, and it indicated you should use the option to stop sensor. I did, and it appeared to fix my problem right away. I now notice that every time I go back into the app, it appears to think there is a started sensor again and I have the option to stop sensor. At first I thought I must accidentally be doing it, but it happens every single time. Should I be doing anything differently? I'm on a Pixel running the latest Android 12 and latest xDrip+. See screenshots. I posted in the FB group, and was asked to post it here. 
<img width="350" src="https://user-images.githubusercontent.com/25916510/154505203-fc78ecb5-7c17-49df-a35b-f75916661adf.jpg" /> <img width="350" src="https://user-images.githubusercontent.com/25916510/154505204-b5305a21-9b15-4e09-9404-7708638c2f2b.jpg" />

**Comment by Navid200** [comment]
@tolot27

**Comment by tolot27** [comment, marked as answer]
As long as it works, every thing is okay. And as long as you get values from Dexcom Share, there is an active sensor, even not at your phone. I wouldn't care about this.

**Comment by mkitchingh** [comment]
Thanks. So do I ever need to 'stop' it? It seemed to clear up the slowness at the time, so I try to make a habit of clicking stop every few weeks or so. I haven't had the slowness/not responding issue again.

**Reply by tolot27** [reply]
No, you don't have to stop the sensor unless you run into your initial issue. At some day if we find time, we will remove this annoying ui issue.
